packages/twenty-website/src/app/_components/ui/layout/header/HeaderDesktop.tsx

```diff
@@ -12,7 +12,19 @@ import {
 } from '@/app/_components/ui/layout/header/styled';
 import { Logo } from '@/app/_components/ui/layout/Logo';
 
-export const HeaderDesktop = () => {
+type Props = {
+  numberOfStars: number;
+};
+
+export const HeaderDesktop = ({ numberOfStars }: Props) => {
+  const formatNumberOfStars = (numberOfStars: number) => {
+    if (numberOfStars < 1000) {
```

Comment:
Hopefully we should never need this 😁.
Makes sens if you put that function in a util (which would be nice, could be not just for stars) but since you kept it in that context it should be as short and focused on that use-case as possible